### PR TITLE
feat: restructure open page

### PR DESCRIPTION
**Pull request by dephraiim**

### CODE REVIEW

**Review by ElTimuro — APPROVED** [review]
- Great, exactly, like I imagined
- I added some text changes as well

LGTM 🚀

**Review by ElTimuro — COMMENTED** [review]
@dephraiim i assume this is due to my local data?
image

**Comment by ElTimuro** [comment]
@dephraiim this is ready to go, but the total signed documents still needs to be added..

**Comment by dephraiim** [comment]
> @dephraiim i assume this is due to my local data?

Yes, occurs during seeding, I fixed it here

https://github.com/documenso/documenso/blob/574cd176c20ac43d5366581f6972bfc44c1442ce/packages/prisma/seed/pr-711-deletion-of-documents.ts#L185

File: apps/marketing/src/app/(marketing)/open/monthly-completed-documents-chart copy.tsx [hunk 1]
```diff
@@ -0,0 +1,57 @@
+'use client';
```
Comment (Mythie) [inline apps/marketing/src/app/(marketing)/open/monthly-completed-documents-chart copy.tsx:1]: `apps/marketing/src/app/(marketing)/open/**monthly-completed-documents-chart copy.tsx**` 💀

File: packages/lib/server-only/user/get-monthly-completed-document.ts [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+  const result = await prisma.$queryRaw<GetCompletedDocumentsMonthlyQueryResult>`
+    SELECT
+      DATE_TRUNC('month', "completedAt") AS "month",
+      COUNT("id") as "count",
+      SUM(COUNT("id")) OVER (ORDER BY DATE_TRUNC('month', "completedAt")) as "cume_count"
+    FROM "Document"
+    WHERE "status" = 'COMPLETED'
+    GROUP BY "month"
+    ORDER BY "month" DESC
+    LIMIT 12
```
Comment (Mythie) [inline packages/lib/server-only/user/get-monthly-completed-document.ts:27]: 🤤

**Pull request merged**